### ISSUE TITLE
Using "Condition" attribute for resource creation

### ISSUE THREAD

**Issue by sampritavh** [seandst/cfn-pyplates#32]
Hi , I am trying to get a template to create an S3 bucket only if a condition is met. Here is my code snippet
````Python
cft.conditions.add(Condition(
		"CreateBucketEnabled",
		{"Fn::Equals" : [{"Ref" : "CreateS3Bucket"}, "True"]}
		)
)
cft.resources.add(Resource(
	'AppS3Bucket',
   'AWS::S3::Bucket',
	 properties = 
   {
      "BucketName": join('-',ref('AWS::StackName'),'-app')
	 },
	c_ref('CreateBucketEnabled')
		 
)
)
````
when I run cfn_py_generate i get the template.json file but I do not see the Condition attribute  in the json file. Am I missing something here ? Can you please help ? 

````json
"Resources": {
    "AppS3Bucket": {
      "Type": "AWS::S3::Bucket",
      "Properties": {
        "BucketName": {
          "Fn::Join": [
            "-",
            [
              {
                "Ref": "AWS::StackName"
              },
              "-app"
            ]
          ]
        }
      }
    },
````

**Comment by seandst** [comment]
This may be the same as #29. If so, try the workaround listed there and let me know if it unblocks you.

In the meantime, the release containing that fix is Coming Soon™.

**Comment by seandst** [comment]
Acting under the assumption that this is indeed a duplicate of #29, it was fixed in https://github.com/seandst/cfn-pyplates/commit/1266a9744fb40be743e822fb554539341e82c1d1 and will be in the next release.
